birdhouse/config/canarie-api/docker_configuration.py.template

```diff
@@ -279,15 +280,36 @@ PLATFORMS = {
 }
 
 CANARIE_MONITORING_EXTRA_CONF_DIR = os.environ.get(
-    'CANARIE_MONITORING_EXTRA_CONF_DIR', '/bogus-notexist')
+    'CANARIE_MONITORING_EXTRA_CONF_DIR', '/bogus-notexist'
+)
+
+
+if sys.version_info[0] == "2":
+    exec_file = execfile  # noqa  # python 2.x
+else:
+    def exec_file(filepath, globals=None, locals=None):
+        """
+        Backward compatible ``execfile`` equivalent for Python 3.x.
+
+        See https://stackoverflow.com/a/41658338/5936364.
+        """
+        if globals is None:
+            globals = {}
+        globals.update({
+            "__file__": filepath,
+            "__name__": "__main__",
+        })
+        with open(filepath, 'rb') as file:
+            exec(compile(file.read(), filepath, 'exec'), globals, locals)
+
 
 if os.path.exists(CANARIE_MONITORING_EXTRA_CONF_DIR):
     # alphabetically sorted for reproducible override precedence
     for extra_conf in sorted(os.listdir(CANARIE_MONITORING_EXTRA_CONF_DIR)):
-        extra_conf_full_path = "{CANARIE_MONITORING_EXTRA_CONF_DIR}/{extra_conf}".format(**locals())
+        extra_conf_full_path = "{}/{}".format(CANARIE_MONITORING_EXTRA_CONF_DIR, extra_conf)
```

Comment:
Why change this?  This is the python2 way to simulate the awesome python3 f-string.  f-strings are way more readable.

Reply:
I can revert, but expanding `locals` on each call instead of just using the variables themselves is not that great. 
Alternatively, we could also do:
```python
"{dir}/{conf}".format(dir=CANARIE_MONITORING_EXTRA_CONF_DIR, conf=extra_conf)
```
or simply 
```python
CANARIE_MONITORING_EXTRA_CONF_DIR + "/" + extra_conf
```

Reply:
`CANARIE_MONITORING_EXTRA_CONF_DIR + "/" + extra_conf` is less bad on the eye yes.

Reply:
Now using Python 3 f-string syntax exclusively.